pkg/apis/app/v1alpha1/app_catalog_types.go

```diff
@@ -46,6 +46,20 @@ func NewAppCatalogCRD() *apiextensionsv1beta1.CustomResourceDefinition {
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// Example of AppCatalog CR as below.
```

Comment:
To make it a valid godoc:

```suggestion
// AppCatalog CR example as below.
```